vsp.go

```diff
@@ -110,6 +114,240 @@ func (v *VSP) PurchaseTickets(ticketCount, expiryBlocks int32, passphrase []byte
 	return nil
 }
 
+func (v *VSP) StartTicketBuyer(balanceToMaintain int64, passphrase []byte) error {
+	if balanceToMaintain < 0 {
+		return errors.New("Negative balance to maintain given")
+	}
+
+	go func() {
+		log.Info("Running ticket buyer")
+		ctx, cancel := context.WithCancel(v.ctx)
+		v.w.cancelAutoTicketBuyer = cancel
```

Comment:
> Why is this a property of the wallet rather than the vsp instance? The vsp instance starts the tb via this `vsp.StartTicketBuyer` method, makes sense for it to also be the one to stop it.

This was made so, so that we can always check the state of the VSP without having to start/initialize a new VSP for every instance were is it to be used.